package.json

```diff
@@ -31,11 +34,15 @@
     "babel-preset-env": "1.6.1",
     "babel-preset-react": "6.24.1",
     "css-loader": "1.0.1",
+    "gulp": "3.9.1",
```

Comment:
There are essentially 2 key types of dependencies you can define in a project:
- dependencies: these will always be installed when running `npm install`
- devDependencies: these will only be installed if `NODE_ENV` is not set to production.

So you don't want to list all modules under "dependencies". Take a look at https://medium.com/@stalonadsl948/dependencies-vs-devdependencies-926e096a3dee to see the difference and how it will help to optimize you final application bundle size.

Reply:
Thanks. Learned something new today. Btw in that article, the author said the following: "By the way, npm install by default installing packages from both dependencies and devDependencies.".  
Doesn't this mean that it will install the dependencies from devDependencies regardless of if we set NODE_ENV to prod or not?

Reply:
Yes it does. By default, `NODE_ENV` is set to `development` if you don't explicitly give it a value. And NPM installs both dependencies as well as devDependencies when NODE_ENV is not set to `production`.

Reply:
Ahh that makes more sense. Thanks for the quick response.